Maestro/subscriptions.json

```diff
@@ -1554,66 +1554,47 @@
         "https://github.com/aspnet/SignalR-Client-Cpp/blob/release/**/*",
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
         "https://github.com/dotnet/aspnetcore/blob/release/2/**/*",
-        // "https://github.com/dotnet/aspnetcore/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
         "https://github.com/dotnet/aspnetcore-tooling/blob/release/2/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
         "https://github.com/dotnet/blazor/blob/release/2/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.1/**/*",
+        "https://github.com/dotnet/blazor/blob/release/3.1/**/*",
         "https://github.com/aspnet/BrowserLink/blob/release/**/*",
         "https://github.com/aspnet/BuildTools/blob/release/**/*",
-        // "https://github.com/dotnet/ef6/blob/release/6.3/**/*",
-        // "https://github.com/dotnet/ef6/blob/release/6.4/**/*",
+        "https://github.com/dotnet/ef6/blob/release/6.4/**/*",
         "https://github.com/dotnet/efcore/blob/release/2/**/*",
-        // "https://github.com/dotnet/efcore/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/efcore/blob/release/3.1/**/*",
+        "https://github.com/dotnet/efcore/blob/release/3.1/**/*",
         "https://github.com/dotnet/extensions/blob/release/2/**/*",
-        // "https://github.com/dotnet/extensions/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/extensions/blob/release/3.1/**/*",
+        "https://github.com/dotnet/extensions/blob/release/3.1/**/*",
         "https://github.com/dotnet/Scaffolding/blob/release/2/**/*",
-        // "https://github.com/dotnet/Scaffolding/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/Scaffolding/blob/release/3.1/**/*",
+        "https://github.com/dotnet/Scaffolding/blob/release/3.1/**/*",
         "https://github.com/aspnet/websdk/blob/release/2/**/*",
-        // "https://github.com/aspnet/websdk/blob/release/3.0/**/*",
-        // "https://github.com/aspnet/websdk/blob/release/3.1/**/*",
+        "https://github.com/aspnet/websdk/blob/release/3.1/**/*",
         "https://github.com/dotnet/cli/blob/release/2/**/*",
-        // "https://github.com/dotnet/cli/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/cli/blob/release/3.1/**/*",
+        "https://github.com/dotnet/cli/blob/release/3.1/**/*",
```

Comment:
@mmitche is this still the right format w/ the 3.1.1xx, 3.1.2xx, 3.1.3xx branches?

Reply:
Yes. You can just imagine that meastro completetely removes `/**/*` and does a StartsWith match.